src/parse_args.ts

```diff
@@ -144,5 +149,9 @@ if (options.capability) {
   }
 }
 
+if (options.suiteParams && !options.params) {
```

Comment:
We can revert to the previous way - https://github.com/vigneshshanmugam/synthetics/blob/b1ba055bee6074ff96f22a4aebda9cfeb22bc362/src/cli.ts#L168-L172